test/dialect/postgresql/test_reflection.py

```diff
@@ -932,6 +932,49 @@ def go():
             ],
         )
 
+    @testing.provide_metadata
+    def test_index_reflection_partial(self):
+        """Reflect the filter defintion on partial indexes
+        """
+
+        metadata = self.metadata
+
+        t1 = Table(
+            "table1",
+            metadata,
+            Column("id", Integer, primary_key=True),
+            Column("name", String(20)),
+            Column("x", Integer),
+        )
+        metadata.create_all()
+        with testing.db.connect().execution_options(autocommit=True) as conn:
+            conn.exec_driver_sql("create index idx1 on table1 ((id || name))")
```

Comment:
**mike bayer** (zzzeek) wrote:

do we need "idx1" in this test?   we don't seem to be testing the expression based portion here and there are other tests for the "skipped unsupported reflection" warning below.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142